change/@fluentui-react-2020-10-05-12-10-48-image-migration-react.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You should remove this file since you're no longer changing anything in the `react` package.